### PR TITLE
move to 1.0

### PR DESCRIPTION
**Pull request by swanderz**

### CODE REVIEW

**Comment by swanderz** [comment]
@jtcohen6 you got a 5 min take on what the error message might be to save me what'd likely be an hour of digging?

```
test/integration/azuresql.dbtspec::test_dbt_empty PASSED                 [ 12%]
test/integration/azuresql.dbtspec::test_dbt_base FAILED                  [ 25%]
test/integration/azuresql.dbtspec::test_dbt_ephemeral FAILED             [ 37%]
test/integration/azuresql.dbtspec::test_dbt_incremental FAILED           [ 50%]
test/integration/azuresql.dbtspec::test_dbt_snapshot_strategy_timestamp FAILED [ 62%]
test/integration/azuresql.dbtspec::test_dbt_snapshot_strategy_check_cols FAILED [ 75%]
test/integration/azuresql.dbtspec::test_dbt_data_test FAILED             [ 87%]
test/integration/azuresql.dbtspec::test_dbt_schema_test FAILED           [100%]

=================================== FAILURES ===================================
____________________________ usecase: test_dbt_base ____________________________
Unknown error: assert 1 == 0
  +1
  -0 in test index 1 (item_type=run_results)
----------------------------- Captured stdout call -----------------------------
Executing step 1/15

Executing step 2/15
_________________________ usecase: test_dbt_ephemeral __________________________
Unknown error: assert 1 == 0
  +1
  -0 in test index 1 (item_type=run_results)
----------------------------- Captured stdout call -----------------------------
Executing step 1/8

Executing step 2/8
________________________ usecase: test_dbt_incremental _________________________
Unknown error: assert 2 == 0
  +2
  -0 in test index 1 (item_type=run_results)
----------------------------- Captured stdout call -----------------------------
Executing step 1/12

Executing step 2/12
________________ usecase: test_dbt_snapshot_strategy_timestamp _________________
Command ['dbt', '--debug', '--no-partial-parse', 'seed', '--target', 'default', '--profile', 'dbt-pytest', '--profiles-dir', '/tmp/tmpz_wwigdo', '--project-dir', '/tmp/tmpz_wwigdo/project', '--vars', "_dbt_random_suffix: '211124001147155631348328'\n"] failed, expected pass! Got rc=2
----------------------------- Captured stdout call -----------------------------
Executing step 1/12

________________ usecase: test_dbt_snapshot_strategy_check_cols ________________
Command ['dbt', '--debug', '--no-partial-parse', 'seed', '--target', 'default', '--profile', 'dbt-pytest', '--profiles-dir', '/tmp/tmpd8r1nw47', '--project-dir', '/tmp/tmpd8r1nw47/project', '--vars', "_dbt_random_suffix: '211124001147155687591709'\n"] failed, expected pass! Got rc=2
----------------------------- Captured stdout call -----------------------------
Executing step 1/20

_________________________ usecase: test_dbt_data_test __________________________
Command ['dbt', '--debug', '--no-partial-parse', 'test', '--target', 'default', '--profile', 'dbt-pytest', '--profiles-dir', '/tmp/tmpp2nd83v3', '--project-dir', '/tmp/tmpp2nd83v3/project', '--vars', "_dbt_random_suffix: '211124001147155745129773'\n"] passed, expected failure! Got rc=0
----------------------------- Captured stdout call -----------------------------
Executing step 1/2

________________________ usecase: test_dbt_schema_test _________________________
Unknown error: assert 1 == 0
  +1
  -0 in test index 2 (item_type=run_results)
----------------------------- Captured stdout call -----------------------------
Executing step 1/5

Executing step 2/5

Executing step 3/5
=========================== short test summary info ============================
FAILED test/integration/azuresql.dbtspec::test_dbt_base
FAILED test/integration/azuresql.dbtspec::test_dbt_ephemeral
FAILED test/integration/azuresql.dbtspec::test_dbt_incremental
FAILED test/integration/azuresql.dbtspec::test_dbt_snapshot_strategy_timestamp
FAILED test/integration/azuresql.dbtspec::test_dbt_snapshot_strategy_check_cols
FAILED test/integration/azuresql.dbtspec::test_dbt_data_test
FAILED test/integration/azuresql.dbtspec::test_dbt_schema_test
========================= 7 failed, 1 passed in 45.69s =========================
ERROR: InvocationError for command '/root/project/.tox/py37/bin/pytest -v test/integration/azuresql.dbtspec' (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py37: commands failed

Exited with code exit status 1

```
```

**Comment by jtcohen6** [comment]
@swanderz Those are the error messages I'd expect to see if you're using v0.5.1, instead of the latest changes from the main branch that are compatible with v1. I'm not sure how to square that with the dev requirement you've specified (`git+`), but I'm pretty sure that's it.

It's on us to push live an updated version of `pytest-dbt-adapter==0.6.0` to make this easier for you (and for us). I'll try to do that this morning.

**Review by jtcohen6 — COMMENTED** [review]
@swanderz Those are the error messages I'd expect to see if you're using v0.5.1, instead of the latest changes from the main branch that are compatible with v1. I'm not sure how to square that with the dev requirement you've specified (`git+`), but I'm pretty sure that's it.

It's on us to push live an updated version of `pytest-dbt-adapter==0.6.0` to make this easier for you (and for us). I'll try to do that this morning.

File: dev_requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 black==20.8b1
-pytest-dbt-adapter~=0.5.1
+git+https://github.com/dbt-labs/dbt-adapter-tests.git#egg=pytest-dbt-adapter
```
Comment (swanderz) [inline dev_requirements.txt:2]:
```suggestion
pytest-dbt-adapter~=0.6.0
```

**Pull request merged**